src/Button.js

```diff
@@ -1,12 +1,21 @@
 import PropTypes from "prop-types";
 import React from "react";
-import { Platform, StyleSheet, Text, TouchableOpacity } from "react-native";
+import {
+  Platform,
+  StyleSheet,
+  Text,
+  TouchableOpacity,
+  PlatformColor,
+} from "react-native";
+import { useTheme } from "./components/hooks";
 
 const COLOR = Platform.OS === "ios" ? "#007ff9" : "#169689";
 
 const DialogButton = (props) => {
   const { label, color, disabled, bold, onPress, style, ...nodeProps } = props;
   const fontWeight = bold ? "600" : "normal";
+  const { styles } = useTheme(__styles);
```

Comment:
Can we rename `__styles` with `buildStyles` to be a bit more descriptive? (a find and replace across the entire codebase should be sufficient 👍)